src/toil_scripts/lib/__init__.py

```diff
@@ -1,4 +1,3 @@
-# FIXME: replace with bd2k.util.iterables.flatten
 def flatten(x):
```

Comment:
Sorry for wasting your time. I forgot that neither concat() nor flatten() are recursive.